core/events/domain/event.py

```diff
@@ -264,6 +272,7 @@ def validate(self, submission: Submission) -> None:
         self._must_be_a_valid_category(submission)
```

Comment:
This is outside the scope of this PR, but want to make a note of this for later:
There are some additional rules that wouldn't be obvious from looking at the classic code (_understatement_). 
* submitters may not choose more than 4 secondary categories in the classic submission workflow. This rule isn't enforced outside that workflow; i.e. admins (and moderators) may elect to add more than four.
* crossing to physics.gen-ph is generally not allowed.
There are others I'm sure I'm missing. We should probably have a separate ticket for reviewing some of these rules.